Source/WebKit/UIProcess/ios/UIKitUtilities.mm

```diff
@@ -202,8 +202,15 @@ - (UIViewController *)_wk_viewControllerForFullScreenPresentation
 
 RetainPtr<UIAlertController> createUIAlertController(NSString *title, NSString *message)
 {
-    auto *alert = [UIAlertController alertControllerWithTitle:title message:message preferredStyle:UIAlertControllerStyleAlert];
-    [alert _setTitleMaximumLineCount:0]; // No limit, we need to make sure the title doesn't get truncated.
+    // FIXME: revisit this once UIKit bug is resolved (see also rdar://101140177).
+    NSMutableAttributedString *alertTitle = [[NSMutableAttributedString alloc] initWithString:title];
+    [alertTitle addAttribute:NSFontAttributeName value:[UIFont preferredFontForTextStyle:UIFontTextStyleHeadline] range:NSMakeRange(0, title.length)];
+
+    auto *alert = [[UIAlertController alloc] init];
+    alert.attributedTitle = alertTitle;
+    alert.title = title;
```

Comment:
Is setting `title` necessary, if we're already setting `attributedTitle`?

Reply:
I am not exactly sure but I think we can keep it there since the idea would probably to remove attributedTitle at some point when the underlying bug is fixed.